### DISCUSSION TITLE
SQANTi3 QC cannot generate classification.txt file

### DISCUSSION THREAD

**Discussion by MR-D-CJ** [ConesaLab/SQANTI3#371] [Q&A] [answered]
When I generate refClassif files for Resuce, I generate ref_classification.txt based on the reference genome, but not the whole file. I used SWANTi3.5.2.2
The code is as follows
`
python /PATH/SQANTI3-5.2.2/sqanti3_qc.py \
/PATH/GRCh38/genecode/gencode.v46.annotation.gtf \
/PATH/GRCh38/genecode/gencode.v46.annotation.gtf \
/PATH/GRCh38/GCA_000001405.15_GRCh38_no_alt/GCA_000001405.15_GRCh38_no_alt_analysis_set.fa \
--CAGE_peak /PATH/refTSS/human.refTSS_v4.1.hg38.space.bed \
--polyA_motif_list /PATH/polyA_info/mouse_and_human.polyA_motif.txt \
-o GRCh38 \
-d /PATH/GRCh38.refTSS4 \
--cpus 20 --report html
`
The log file is as follows
<img width="1347" alt="截屏2024-12-21 10 19 47" src="https://github.com/user-attachments/assets/16754efc-9993-4ddf-a587-b28d4dcf92dc" />
<img width="1847" alt="截屏2024-12-21 10 43 54" src="https://github.com/user-attachments/assets/01389f16-8d22-4b5e-aa91-088d976b0518" />

**Comment by pabloati** [comment, marked as answer]
Hi, could you try to run SQANTI3 in its latest release? It has been updated to the release 5.3.6

Best,
Pablo
